DESCRIPTION

```diff
@@ -72,36 +72,38 @@ Imports:
   withr (>= 2.4.0),
```

Comment:
This would be a substantial contribution if merged, so feel free to add yourself as a contributor (`role = "ctb"`) in the DESCRIPTION file.

Reply:
Exciting! It would be my first OSS contribution to an R package — and I'm quite keen to put together something clean and useful.